docker/go/nethttp/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:latest
+FROM golang:1.8
```

Comment:
Why Go 1.8 and no any other?

Reply:
it is the minimum version required, it is defined in the Agent Go README